### PR TITLE
Fixes #465 :Added background image to a blank section, and fixed dead-end links

### LINKED ISSUE #465: Dead-end links and blank background in a section
### Expected results
Working links
### Actual results
Dead-end links

### Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #465

### CODE REVIEW

**Review by sayak119 — COMMENTED** [review]
Can you provide a demo link

**Comment by saarthakchats** [comment]
@sayak119 https://saarthakchats.github.io/clipsclopgci17.fossasia.org/

**Comment by saarthakchats** [comment]
@sayak119 Can you approve?

**Review by sayak119 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -451,7 +451,7 @@ <h1 class="bold mb0">FOSSASIA <br>Based in Asia.</h1>
 	</div>
 
 	<div class="align-bottom text-center">
-		<a class="btn btn-white mb32" href="#">Pitch Us</a>
+		<a class="btn btn-white mb32" href="2017.fossasia.org">What happened in the last Summit?</a>
```
Comment (NaviRocker) [inline index.html]: Add `https://2017.fossasia.org` instead of `2017.fossasia.org`

**Review by NaviRocker — CHANGES_REQUESTED** [review]

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@saarthakchats  Please do the changes suggested by @naveentherocker  👍

**Comment by saarthakchats** [comment]
@ParthS007 
Done

**Comment by ParthS007** [comment]
@saarthakchats  Can you remove the social link below `MEET US IN 2018`, I think It is no needed there 👍  else looks good to me

**Comment by saarthakchats** [comment]
@ParthS007 
Done!
Please merge and also approve my task https://codein.withgoogle.com/dashboard/

**Review by ParthS007 — APPROVED** [review]

**Comment by ParthS007** [comment]
@saarthakchats  Please give link of your task instance

**Comment by saarthakchats** [comment]
Its is approved: by harshit prasad! thanks!

**Pull request merged**